jfrog-cli/artifactory/utils/utils_test.go

```diff
@@ -23,24 +23,64 @@ func TestGetHomeDir(t *testing.T) {
 }
 
 func TestRemoveCredentialsFromURL(t *testing.T) {
```

Comment:
This comment applies to both the *TestRemoveCredentialsFromURL* and * TestReturnErrorOnNotFound* functions:
In case one of the tests fails, I'm not sure it will be easy to understand from the test error message, which line pattern failed. What do you think?

Reply:
I've changed the tests to also provide the regex result and also the output of the line after masking.